packages/checkout/sdk/src/widgets/definitions/parameters/sale.ts

```diff
@@ -16,8 +15,6 @@ export type SaleWidgetParams = {
   amount?: string;
   /** Environment id from Immutable Hub */
   environmentId?: string;
-  /** Contract address of the token accepted for payments */
-  fromTokenAddress?: string;
```

Comment:
@mimi-imtbl just a note to remember to update the docs.
https://github.com/immutable/docs/pull/805

Reply:
done